public/index.html

```diff
@@ -28,7 +28,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>screw your neighbour - 36</title>
```

Comment:
I would just write "Screw Your Neighbour"

Reply:
Yes, fine! I thought that the group number should we add but I not have too.